src/cluster.h

```diff
@@ -17,6 +17,7 @@
 #define CLUSTER_FAIL_UNDO_TIME_MULT 2 /* Undo fail if master is back. */
 #define CLUSTER_FAIL_UNDO_TIME_ADD 10 /* Some additional time. */
 #define CLUSTER_FAILOVER_DELAY 5 /* Seconds */
+#define CLUSTER_DEFAULT_ALLOW_REPLICA_MIGRATION 1 /* Allow automatic replica migrations by default. */
```

Comment:
This isn't used. We used to put defaults in .h files, but these days we just keep them in config.c

Reply:
fixed